tensorflow/python/autograph/converters/logical_expressions_test.py

```diff
@@ -59,6 +59,19 @@ def f(a, b, c, d):
     self.assertTrue(self.evaluate(tr(constant_op.constant(1), 2, 2, 1)))
     self.assertFalse(self.evaluate(tr(constant_op.constant(1), 2, 2, 3)))
 
+  def test_comparison_repeated(self):
```

Comment:
This one is going to fail cause is missing `ag__.ld` for the `constant_op` (b). Let me know if we could modify it with a workaround or we could remove it.

Reply:
This need to produce:
https://github.com/tensorflow/tensorflow/issues/55278#issuecomment-1073132318

Reply:
Actually, I think this test is redundant, see the test just above it. Sorry I missed it originally! The comment also explains the failure.

Reply:
Ok I've removed the test. Let's do this to the close source tests.